elixir_phoenix/priv/repo/migrations/20150930050801_create_client.exs

```diff
@@ -0,0 +1,22 @@
+defmodule EboshiApiShootoutElixirPhoenix.Repo.Migrations.CreateClient do
+  use Ecto.Migration
+
+  def change do
+    create table(:clients) do
+      add :name, :string
+      add :address, :string
+      add :city, :string
+      add :state, :string
+      add :zip, :string
+      add :country, :string
+      add :email, :string
+      add :contact, :string
+      add :phone, :string
+      add :created_at, :datetime
+      add :updated_at, :datetime
+
+      timestamps
```

Comment:
ecto uses `inserted_at` and `updated_at` so it does not line up quite like your schema.  it doesn't look like there is an out of the box way to change `insterted_at` to `created_at`  - so ,yes, this migration is a little borky.  ends up not mattering right now since we are creating the db schema via the tests.

I will have to deal with it when we get to inserts and roll my own solution to get it in `created_at` - but i decided to punt until then!